connectivity/FEATURE_BLE/source/generic/PrivateAddressController.cpp

```diff
@@ -129,22 +131,32 @@ void PrivateAddressController::generate_non_resolvable_address()
 
 bool PrivateAddressController::is_controller_privacy_supported()
 {
+#if BLE_GAP_HOST_BASED_PRIVACY
+    return false;
```

Comment:
I'm not sure this is accurate host based privacy may be enabled on a board with LL privacy available.

Reply:
narrowed down the meaning to address resolution and fixed ifdefs accordingly